association_backend/association/api/permissions.py

```diff
@@ -0,0 +1,10 @@
+from association.api.context import Info
+from association.settings import JWT_USERS_COOKIE_NAME
+from strawberry.permission import BasePermission
+
+
+class IsJWTAvailable(BasePermission):
+    message = "JWT Not Available"
+
+    def has_permission(self, source, info: Info, **kwargs):
+        return info.context.request.cookies.get(JWT_USERS_COOKIE_NAME, None) is not None
```

Comment:
hmm if you want to see how you will receive the pastaporto: SubscriptionResponse.from_domain(subscription)

Reply:
I don't understand
Could you be clearer?

Reply:
Ahah I think I copied and pasted the wrong thing :) I meant you can see this PR: https://github.com/pythonitalia/pycon/pull/2084/files

Reply:
ok I have marked it
I have removed `IsJWTAvailable`